src/site/markdown/server/manage-webadmin.md

```diff
@@ -1868,6 +1869,118 @@ Response codes:
  - 404: The domain does not exist.
  - 500: Internal error
 
+## Administrating Sieve quotas
+
+Some limitations on space Users Sieve script can occupy can be configured by default, and overridden by user.
+
+ - [Retieving global sieve quota](#Retieving_global_sieve_quota)
+ - [Updating global sieve quota](#Updating_global_sieve_quota)
+ - [Removing global sieve quota](#Removing_global_sieve_quota)
+ - [Retieving user sieve quota](#Retieving_user_sieve_quota)
+ - [Updating user sieve quota](#Updating_user_sieve_quota)
+ - [Removing user sieve quota](#Removing_user_sieve_quota)
+
+### Retieving global sieve quota
+
+This endpoints allow to retrieve the global Sieve quota, which will be users default:
+
+```
+curl -XGET http://ip:port/sieve/quota
+```
+
+Will return the bytes count allowed by user per default on this server.
+
+```
+102400
+```
+
+Response codes:
+ - 200: Request is a success and the value is returned
+ - 404: No quota is being configured
+ - 500: Internal error! Something went bad on James side.
```

Comment:
Do we need to document 500 ?

Reply:
I'd say yes, it's done on other endpoints...

Reply:
then why having removing them?

Reply:
We may add a `All endpoints might returns 500 error` on the top of the document to avoid this information duplication, that is not route specific, no?

Reply:
It's like explaining HTTP, what's the point ?